Automattic/wp-calypso
# Gutenboarding: Show theme default fonts

#### Changes proposed in this Pull Request

* Show design default fonts in the fonts list

Closes #40560

#### Testing instructions

* /gutenboarding
* Verify the theme default fonts are shown 1st in the list
* Verify font selection works
